src/database/db_build.js

```diff
@@ -0,0 +1,13 @@
+
+const fs = require('fs');
+const db_connection = require('./db_connection');
+const build_file = fs.readFileSync(__dirname + '/db_build.sql','utf8');
+db_connection.query(build_file,(error,result) => {
```

Comment:
You're not using result here so it isn't necessary to pass in it in here

Reply:
i remove result